com.github.ImCompressor.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "com.github.ImCompressor",
```

Comment:
this should probably be com.github.huluti.ImCompressor

Reply:
Thank's! Fixed in https://github.com/Huluti/ImCompressor/commit/9f3bc2e633933fae2409df384e28b31244eeb80b.